### PR TITLE
cloud sync blog post

### PR DESCRIPTION
**Pull request by eanders-MS**
First blog post since joining the team! Anything missing or poorly worded? Don't go easy on me! :) Would love feedback, and feel free to edit as you see fit.

### CODE REVIEW

**Review by Jaqster — APPROVED** [review]
Looks good Eric!  I feel like this is a pretty well-known paradigm, so doesn't need too much explanation.  Maybe link to https://makecode.com/privacy when you talk about privacy?  This just points to Microsoft's Privacy Policy, but maybe good to call out.

File: docs/blog.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Microsoft MakeCode Blog
 
+## [Introducing Cloud Sync for MakeCode Arcade](/blog/arcade/intro-cloud-sync)
+
+August 12th, 2021 by [eanders-ms](https://github.com/eanders-ms)
+
+Over the past year we've been working to make it possible to sign into MakeCode with a [Microsoft Account](https://aka.ms/AAdd6f8). When you sign into MakeCode, you make it possible for us to remember who you are, and this unlocks a bunch of exciting new possibilities.
```
Comment (ElizabethRiffle) [inline docs/blog.md:7]: Here are my edits:

Over the past year we've been working hard to bring you the ability to sign into MakeCode with a [Microsoft Account](https://aka.ms/AAdd6f8). When you sign into MakeCode, you make it possible for us to remember who you, and this unlocks a bunch of exciting new possibilities.

File: docs/blog/arcade/intro-cloud-sync.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Introducing Cloud Sync for MakeCode Arcade
+
+**Posted on August 12th, 2021 by [eanders-ms](https://github.com/eanders-ms)**
+
+**Note**: As of this writing, sign-in and cloud sync are only available in [MakeCode Arcade](https://arcade.makecode.com).
+
+Over the past year we've been working to make it possible to sign into MakeCode with a [Microsoft Account](https://aka.ms/AAdd6f8). When you sign into MakeCode, you make it possible for us to remember who you are, and this unlocks a bunch of exciting new possibilities. For example, it becomes possible for us to remember what you were working on and make your projects available wherever you are. In fact this is the first feature we've built for our signed in users, and we're calling it **Cloud Sync**.
+
```
Comment (ElizabethRiffle) [inline docs/blog/arcade/intro-cloud-sync.md:8]: Change this to match the language above

I would also change it to either:
- "make your projects available wherever you are on any device" or "make your projects available on any device you use" (i just think the any device should be emphasized)

File: docs/blog/arcade/intro-cloud-sync.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## What is Cloud Sync?
+
+Have you ever been working on a MakeCode project at school and wanted to take it home? In the past, users have found creative ways to move projects from device to device and browser to browser, and these methods will still work! But Cloud Sync makes this a seamless experience: when you sign into MakeCode, your cloud-saved projects are download to you automatically.
+
+Perhaps you've had the experience of returning to a lab computer only to realize the machine had been reset overnight, erasing your projects with it. Losing all your work can be devastating. Cloud Sync guards against this by continouously saving your work to the cloud as you edit your project.
+
+## Nice! How do I enable it?
+
+Cloud sync is enabled automatically when you sign into MakeCode. After signing in, your existing projects remain local, meaning they're not automatically transferred to the cloud all at once. To transfer an existing project to the cloud, just open it in the editor. The editor's auto-save feature will trigger a cloud save and voilà, your project is in the cloud. Look for the editor's new cloud save indicator:
```
Comment (ElizabethRiffle) [inline docs/blog/arcade/intro-cloud-sync.md:17]: Might be good to add that when you start a new project when signed it automatically is saved to the cloud

**Review by ElizabethRiffle — APPROVED** [review]

File: docs/blog/arcade/intro-cloud-sync.md (hunk 2 above)
Comment (eanders-MS) [reply docs/blog/arcade/intro-cloud-sync.md:8]: How about "wherever you use MakeCode"?

**Pull request merged**